checks/trunk-outdated-bindgen/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "trunk"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
```

Comment:
Should this pin `wasm-bindgen` to 0.2.85 so that we can test that things keep working?

Otherwise the next time we run `cargo update` in here it might break the test

Reply:
Done :+1: